main.go

```
@@ -183,6 +189,8 @@ func PostImage(w http.ResponseWriter, r *http.Request) {
 	}
 
 	fmt.Fprintf(w, "Successfully saved: %s", filePath)
+
+	w.Write([]byte(filepath.Join(Config.Domain, filePath)))
```

golangci-lint

Error return value of `w.Write` is not checked

(errcheck)